### ISSUE TITLE
Question about some categories set special train parameters

### ISSUE THREAD

**Issue by husuqi** [pangdatangtt/UniNet#7]
Thank you for developing and maintaining such an excellent framework! But I have some questions about the following code：
model.py: `T = 0.1 if self._class_ in ['transistor', 'pill', 'cable', 'bottle', "grid", 'foam'] else self.T`;
`    def feature_selection(self, b, a, max):
        if self._class_ in ['transistor']:
            return a`
train_unsupervisedAD.py: ` optimizer1 = torch.optim.AdamW(list(Target_teacher.parameters()), lr=1e-4 if c._class_ == 'transistor' else c.lr_t,
                                   betas=(0.9, 0.999), weight_decay=1e-5)`
In what circumstances would such special settings（eg. transistor）be necessary for the code above?

**Comment by pangdatangtt** [comment]
Thanks for your interest. 

Similar to the baseline model, our method also faces the issue of training instability on some categories. In our code,  factors like temperature coefficient or learning rate would affect the performance. We thus tested multiple settings and then selected the optimal setting.

**Comment by husuqi** [comment]
Thank you for your reply! Is the use of DFS also the best setting to choose based on the experiment？

**Comment by pangdatangtt** [comment]
Yes, only transistor category would be affected.
